.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
As I commented in https://github.com/strongloop/loopback-next/pull/503#issuecomment-322129628, in order to disable package-lock, we should disable it at npm level in a `.npmrc` file.

>  I think adding `package-lock.json` to `.gitignore` will not work well - you will still have those lock files around, and therefore `npm` will keep installing locked dependencies. As a result, you can end up in a situation where `npm test` is passing locally on your machine, but fails on CI servers.

> IMO, the correct solution is to disable package-lock in a project-level [npmrc](https://docs.npmjs.com/files/npmrc):

```
package-lock=false
```

> That way no package-lock files are created at all, and there is nothing to git-ignore.